tests/Integrations/Laravel/APIGWTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace DDTrace\Tests\Integrations\Laravel;
+
+use DDTrace\Tests\Common\WebFrameworkTestCase;
+use DDTrace\Tests\Frameworks\Util\Request\GetSpec;
+
+
+class APIGWTest extends WebFrameworkTestCase
+{
+    public static $database = "laravel11";
+
+    public static function getAppIndexScript()
+    {
+        return __DIR__ . '/../../Frameworks/Laravel/Latest/public/index.php';
+    }
+
+    protected static function getEnvs()
+    {
+        return array_merge(parent::getEnvs(), [
+            'APP_NAME' => 'laravel_test_app',
+            'DD_SERVICE' => 'my_service',
+            'DD_ENV' => 'local-test',
+            'DD_VERSION' => '1.0',
+            'DD_TRACE_INFERRED_PROXY_SERVICES_ENABLED' => 'true',
+            //'DD_TRACE_HEADER_TAGS' => 'x-dd-proxy-domain-name,x-dd-proxy,x-dd-proxy-httpmethod,x-dd-proxy-path,x-dd-proxy-request-time-ms,x-dd-proxy-stage',
+        ]);
+    }
+
+    public function testInferredProxy()
+    {
+        $this->tracesFromWebRequestSnapshot(function () {
+            $this->call(
+                GetSpec::create(
+                    'A simple GET request returning a string',
+                    '/simple?key=value&pwd=should_redact',
+                    [
+                        'x-dd-proxy: aws.apigateway',
```

Comment:
The way we implemented it in other languages was to have a map w/the header value of `aws-apigateway` to map to a span name value of `aws.apigateway`. We want the header value to remain unchanged and we want the span name to be slightly different which can be represented by a key value map.

That way we can add more values in the future if we have more proxy headers that correspond to different operation names

Reply:
Refer to this as an [example](https://github.com/DataDog/dd-trace-go/blob/fb51babc4beed2e0037c41df97848313e7e43af3/contrib/internal/httptrace/inferred_proxy.go#L74)